README.md

```diff
@@ -33,7 +18,10 @@ It is generally good practice to separate the view (presentation/ aesthetics) an
 <h1>{{title}}</h1>
 <p>{{description}}</p>
 ```
-- Handlebars can be used for both client-side and server-side rendering.
+- Handlebars can be used for both client-side and server-side *rendering*.
+
+**What is rendering?**
+Rendering generally means generating an image from code. In the context of express / handlebars it is slightly different. When we use the [```render```](http://expressjs.com/en/api.html#res.render) function in express, we are telling  express to use the rendering engine (in our case handlebars) to take various pieces of input and compose them into an html string to be sent to the client. This wont actually be turned into an image until it hits the browser.
```

Comment:
- is it possible to explain render without using the word _rendering_
- This wont (won't) actually